test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Models/LibraryIdViewModelTests.cs

```diff
@@ -18,7 +18,17 @@ public class LibraryIdViewModelTests
         private readonly ILibraryCatalog _testCatalog = new LibraryCatalog()
```

Comment:
Unless the additional data being added is relevant to all of the tests, I don't like having it all in one place (the original dataset was shared because it was broadly applicable).

The reason for this is that a unit test should contain the minimal set of data to express the scenario.  This also makes it easier to tell during debugging which part of the test data is relevant to the issue (ideally, all of it); otherwise, you might have to spend time debugging if it's because of the "garbage data", when that data might not be relevant to the failure or scenario being tested at all.

What if we changed these tests to each create the catalog with data they need, then pass that to a helper factory method to get the LibraryIdViewModel to test?  Something along the lines of:
```
var stubCatalog = new LibraryCatalog()
                      .AddLibrary(new Library { Name = "test", Version = "1.0" })
                      .AddLibrary(new Library { Name = "@scope/test", Version = "1.2" } );
LibraryIdViewModel testObj = CreateLibraryIdViewModel(stubCatalog);
```

Reply:
Resolved in commit 2